src/inmanta/graphql/graphql.py

```diff
@@ -0,0 +1,30 @@
+"""
+Copyright 2025 Inmanta
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+Contact: code@inmanta.com
+"""
+
+from typing import Any
+
+import inmanta.graphql.schema
+from inmanta.protocol import methods_v2
+from inmanta.protocol.decorators import handle
+from inmanta.server import SLICE_GRAPHQL, protocol
+
+
+class GraphQLSlice(protocol.ServerSlice):
+
+    def __init__(self) -> None:
+        super().__init__(name=SLICE_GRAPHQL)
+
+    @handle(methods_v2.graphql)
+    async def graphql(self, query: str) -> Any:  # Actual return type: strawberry.types.execution.ExecutionResult
+        return await inmanta.graphql.schema.get_schema().execute(query)
```

Comment:
We may want to add make a ticket to integrate this better, so we can also expose the schema

Reply:
I'm not sure of how it would look like, do you have any suggestion?

Reply:
I would make it work similar to what the example server for graphql does to serve its schema

Reply:
https://app.zenhub.com/workspaces/core-5a7c152b04a1652024289b7b/issues/gh/inmanta/inmanta-core/8820